### ISSUE TITLE
not working on CE 1.9.3.7

### ISSUE THREAD

**Issue by paterlinimatias** [progammer-rkt/Rkt_SbCache#9]
If I turn on cache for blocks html output and navigate two different categories with subcategories inside, the second category I visit shows the subcategories of the first category. Problem goes away if I turn off the cache.

If you have any advice on how to solve this issue, it'll be highly appreciated.

**Comment by jgilbert99** [comment]
I have exactly the same issue. Does anyone have a solution for this?

**Comment by rajeev-k-tomy** [comment]
Could you please provide more details such as:

1. Is layered navigation enabled for the categories in the context?
2. Does this issue exist if this extension is disabled?
3. Do you use any other cache extensions? Which type cache you are using?

I tried to replicate the given scenario, however, it is working properly.

**Comment by jgilbert99** [comment]
Yes, layered navigation is enabled. Have you tested it with layered navigation enabled?
